recipes/seasocks/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
@uilianries should hooks have failed? https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h048-cmake-version-required

test package says 3.3

Reply:
3.3 > 2.8, so aren't we good?

Btw, the hook only tests for a version starting with 2, so requiring 3.0 would also succeed.
See https://github.com/conan-io/hooks/blob/20cfaaa17773739184af534a4a48821e53a2c914/hooks/conan-center.py#L471

imho, it would be better to use a regex (everything looks better with a regex :innocent: :smiling_imp: )
```python
for m in re.finditer(r"cmake_minimum_required\s*\(\s*version\s+([0-9.]+)', cmake_content, flags=re.I):
    try:
        if tools.Version(m.group(1)) >= "3.1":
            out.error("version too low")
    except ValueError:
        out.error("Invalid minimum cmake version")
```

Reply:
It says should require 3.1 by default, the normative language is not very strong okay. 

maybe the hook should warn if that's not the default ?

Reply:
A warn by default for too high cmake requirements would be too strict, because there might exist reasons to test newer cmake features. Imho.

Looking at this recipe's test_package cmakelists, it should build on cmake 3.1. I don't think it uses more recent features.

Reply:
Hi folks. I'm not sure how to unblock this. Is there something I can do to help move this along?

Reply:
I think we're all good here.
We only need 2 more reviews from @SSE4 and @danimtb

Reply:
The CCI team is very busy working hard 👏

Reply:
Indeed, but sometimes a poking stick does wonders :smiling_imp:

Reply:
Thanks all! I didn't understand the last few comments fully and so was just making sure there wasn't something on my side to look in to! Appreciate all the work!